uvicorn/main.py

```diff
@@ -478,7 +478,7 @@ def run(
     reload_delay: float = 0.25,
     workers: typing.Optional[int] = None,
     env_file: typing.Optional[str] = None,
-    log_config: typing.Optional[typing.Union[dict, str]] = None,
+    log_config: typing.Optional[typing.Union[dict, str]] = LOGGING_CONFIG,
```

Comment:
```suggestion
    log_config: typing.Optional[typing.Union[typing.Mapping[str, Any], str]] = LOGGING_CONFIG,
```

At least this way we'll get an error from the type checker if we try to mutate the default argument.

Reply:
I'm going to merge https://github.com/encode/uvicorn/pull/1539 after this PR.

Reply:
But it wouldn't make any difference for here...